packages/redux-dynamic-modules-observable/src/EpicManager.ts

```diff
@@ -1,7 +1,7 @@
 import { getObjectRefCounter, IItemManager } from "redux-dynamic-modules-core";
 import { Epic, ofType, EpicMiddleware } from "redux-observable";
```

Comment:
This import is now unused

```suggestion
import { Epic, EpicMiddleware } from "redux-observable";
```